react/features/notifications/components/Notification.web.js

```diff
@@ -1,29 +1,72 @@
 import Flag from '@atlaskit/flag';
 import EditorInfoIcon from '@atlaskit/icon/glyph/editor/info';
 import PropTypes from 'prop-types';
+import ErrorIcon from '@atlaskit/icon/glyph/error';
+import WarningIcon from '@atlaskit/icon/glyph/warning';
+import { colors } from '@atlaskit/theme';
 import React, { Component } from 'react';
 
 import { translate } from '../../base/i18n';
 
+declare var interfaceConfig: Object;
+
+/**
+ * Secondary colors for notification icons.
+ *
+ * @type {{error, info, normal, success, warning}}
+ */
+const ICON_COLOR = {
+    error: colors.R400,
+    info: colors.N500,
+    normal: colors.N0,
+    success: colors.G400,
+    warning: colors.Y200
+};
```

Comment:
So...here's the ugly part that I caused. There's another notification component that has a toggle. Do you think with these set of changes it's still okay for that component to exist?

Reply:
You're right. I can make it import the Notification component and use it by extending it or I can add the toggle as a prop in the Notification.web.js. WDYT will be better?

Reply:
I feel like wrapping/extending the existing component goes along more with the design of being declarative and using components as building blocks. I'm not sure how easy it would be to add the toggling behavior, though, because I leveraged the ability of Flag to take anything as a description and display it, so I pass in a ReactElement with a toggle... If you see a path to resolving my mistakes and it's really easy then please go ahead but I think another options is to plan for follow-up changes that you or I do if they seem possible to do.